docs/workflow/requirements/macos-requirements.md

```diff
@@ -1,39 +1,37 @@
-Requirements to build dotnet/runtime on macOS
-=====================
+# Requirements to build dotnet/runtime on MacOS <!-- omit in toc -->
 
-This guide will walk you through the requirements needed to build dotnet/runtime on macOS. We'll start by showing how to set up your environment from scratch.
+* [Environment](#environment)
+  * [Xcode](#xcode)
+  * [Toolchain Setup](#toolchain-setup)
 
-Environment
-===========
+This guide will walk you through the requirements needed to build _dotnet/runtime_ on MacOS. We'll start by showing how to set up your environment from scratch.
 
-These instructions were validated on macOS 10.15 (Catalina).
+## Environment
 
-Xcode
------
+These instructions were tested on MacOS 12.5.1 (Monterey).
```

Comment:
I do not think that these notes are useful. If somebody is reading this doc, they will most likely not going to have the same version of the OS. This note is just going to scare them.

Note that the macOS version is not the only variable in these instructions. XCode version

Reply:
(same for other OSes)

Reply:
Those notes already existed so I just followed. If we can agree that they are not necessary, I can remove them. We should remove them from everywhere though, not just the MacOS doc.

Reply:
Agree. My vote would be to delete them from everywhere.